syserol/regression.py

```diff
@@ -36,52 +36,43 @@ def function_elastic_net(function="ADCC"):
     df_variables = df_merged.drop(["subject"] + functions, axis=1)
 
     # perform regression
-    Y_pred = elasticNetFunc(df_variables, Y)
-
+    enet = elasticNetFunc(df_variables, Y)
```

Comment:
elasticNetFunc() isn't really a simplification anymore. Just copy the relevant lines into this function.